File: src/main/java/ar/edu/unq/epers/unidad4/ServletInitializer.java
package ar.edu.unq.epers.unidad4;

import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;

public class ServletInitializer extends SpringBootServletInitializer {

    @Override
    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
        return application.sources(Unidad4EjemploNeo4jSpringApplication.class);
    }

}

File: src/main/kotlin/ar/edu/unq/epers/unidad5/ServletInitializer.kt
package ar.edu.unq.epers.unidad5

import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer

class ServletInitializer : SpringBootServletInitializer() {

    override fun configure(application: SpringApplicationBuilder): SpringApplicationBuilder {
        return application.sources(Unidad5EjemploNeo4jSpringApplication::class.java)
    }

}
